indsl/sustainability/co2_emissions_calculations.py

```diff
@@ -29,12 +29,12 @@ def emissions_factor_combustor(
     * Method 3: Specify the carbon content of the fuel and use that as the emissions factor.
 
     Args:
-        emissions_factor: CO2 emitted either per mass, volume or energy unit.
-        heating_value: This can be either the lower or higher heating value depending on the country's legal requirements.
-        carbon_content: Carbon content of fuel (mass of carbon by mass or volume of fuel).
+        emissions_factor (float): CO2 emitted either per mass (e.g., kg CO2/kg fuel), volume (e.g., kg CO2/m^3 fuel), or energy unit (e.g., kg CO2/MJ).
```

Comment:
```suggestion
        emissions_factor: CO2 emitted either per mass (e.g., kg CO2/kg fuel), volume (e.g., kg CO2/m^3 fuel), or energy unit (e.g., kg CO2/MJ).
```

We should not specify types here and other cases bellow. The data types are already provided in the function definition and will be handled in the UI accordingly.